examples/high_level_functions_special_objects.js

```diff
@@ -0,0 +1,121 @@
+var t1 = new Date().getTime();
```

Comment:
Please move this file to folder high_level_functions_demos and rename it - add _demo at the end